### PR TITLE
Migrate certain unit variables to passVars in new chains

### PR DESCRIPTION
**Pull request by mistydemeo**
Certain chains need to be able to inherit passVars set in previous chains. An example of this is identification, where a second pass of identification happens after extraction during the transfer; the previous choice made needs to be able to apply.

Unit variables with the variable name `replacementDict` will now have their values included into passVars at the creation of every subsequent chain in the unit.

### CODE REVIEW

File: src/MCPServer/lib/jobChain.py [hunk 1]
```diff
@@ -33,6 +34,21 @@
 #
 #potentialToHold/getFromDB
 #-previous chain links
+def fetchUnitVariableForUnit(unit_uuid):
+    "Returns a dict combining all of the replacementDict unit variables for the specified unit."
```
Comment (Hwesta) [inline src/MCPServer/lib/jobChain.py]: Triple quotes around the docstring?

Comment (mistydemeo) [inline src/MCPServer/lib/jobChain.py]: Is that the preferred style for single-line docstrings?

**Comment by Hwesta** [comment]
Looks good, I could only find the most minor of nitpicks.  :shipit:

File: src/MCPServer/lib/jobChain.py (hunk 1 above)
Comment (mistydemeo) [inline src/MCPServer/lib/jobChain.py]: > Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.

http://www.python.org/dev/peps/pep-0257/

Looks like that is preferred, will fix.

File: src/MCPClient/lib/clientScripts/identifyFileFormat.py [hunk 2]
```diff
@@ -12,9 +12,29 @@
     sys.path.append(path)
 os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.common'
 from fpr.models import IDCommand, IDRule, FormatVersion
-from main.models import FileFormatVersion, File
+from main.models import FileFormatVersion, File, UnitVariable
 
 
+def save_idtool(file_, value):
+    """
```
Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/identifyFileFormat.py:19]: @Hwesta I like the style of starting the comment on the line after the triple quotes, does that work for you?

Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/identifyFileFormat.py:19]: Sure.

While we're talking docstring minutia, there should be a newline after the summary. 

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

**Comment by mistydemeo** [comment]
Pushed!

**Pull request closed without merging**